docs/release-notes.asciidoc

```diff
@@ -13,6 +13,74 @@ This section summarizes the changes in each release.
+* Adds a date time picker to the *Threat Intel* tab that allows users to query the alert for indicator matches from a specific time range ({pull}107234[#107234]).
+* Adds memory threat protection as an option for configuring integration policies. This option detects and stops in-memory threats on Windows systems, such as shellcode injection, which are used to evade traditional file-based detection techniques. Users can also make exceptions for memory protection alerts ({pull}102196[#102196], {pull}101365[#101365]).
+* Adds malicious behavior protection as an option for configuring integration policies. This option detects and stops threats by monitoring the behavior of system processes for suspicious activity. Behavioral signals are much more difficult for adversaries to evade than traditional file-based detection techniques. Users can also create exceptions for behavior protection alerts ({pull}106853[#106853], {pull}106247[#106247]).
+* For {stack} version >= 7.15.0, host isolation is supported for endpoints in Windows, macOS, and these Linux distributions ({pull}108230[#108230]):
+
+** CentOS 7/Rehl 7
+** Ubuntu 18.04
+** Ubuntu 20.04
+** AWS Linux 2
+
+* Implement the accordion view on the *Threat Intel* tab so that threat matches and their details can be easily viewed ({pull}106609[#106609]).
+* Improves the event enrichment query performance ({pull}106150[#106150]).
+* Allows users to select a custom date range when exploring their endpoint’s activity log ({pull}104085[#104085]).
+* Adds advanced policy keys for memory signature and shellcode protection ({pull}101721[#101721]).
+
+[discrete]
+[[bug-fixes-7.15.0]]
+==== Bug fixes and enhancements
+* Allows users to bulk close exceptions to close acknowledged alerts ({pull}110147[#110147]).
+* Removes missing fields from the Trend histogram and Count table on the Alerts page ({pull}108843[#108843]).
+* Updates ECS 1.11 signal mappings ({pull}108764[#108764]).
+* Adds the `operatorsList` property in exceptions builder, which allows users to define options in the operators list ({pull}108015[#108015]).
+* Updates alerts enriched with threat intelligence data to use the latest ECS threat fields ({pull}107988[#107988]).
+* Highlights building block alerts in the Alerts table ({pull}107727[#107727]).
+* Updates MITRE ATT&CK mappings to v9.0 ({pull}107708[#107708]).
+* Makes improvements to the Timeline title and moves the Timeline description to the *Notes* tab ({pull}106544[#106544]).
+* Adds more actions to the *Take action* menu in the Alert details flyout ({pull}105767[#105767]).
+* Fixes the wrong nested package object assignation in the policy delete response ({pull}110824[#110824]).
+* Fixes a bug where parts of the *Activity Log* tab are loaded twice because data was being fetched twice ({pull}110233[#110233]).
+* Removes the clear field button (*x*) within the date time picker on the *Activity Log* tab ({pull}110035[#110035]).
```

Comment:
I think I prefer "date _and_ time picker." But as you noted in another PR, this is a term that we probably want to have an agreed-upon standard usage.